src/components/ebay-notice/index.js

```diff
@@ -16,30 +18,65 @@ const constants = {
 
 const defaults = {
     type: 'page',
-    status: 'confirmation'
+    status: 'priority',
+    hidden: false,
+    dismissible: false
 };
 
-function getTemplateData(state, input) {
+function getInitialState(input) {
+    const hidden = defaults.hidden;
     const type = input.type || defaults.type;
     const headingTag = (type === 'page' && input.headingLevel) ? `h${input.headingLevel}` : constants[type].headingTag;
     const status = input.status || defaults.status;
+    const dismissible = (input.dismissible && type === 'page') || defaults.dismissible;
 
     return {
         mainTag: constants[type].mainTag,
         headingTag: headingTag,
         contentTag: constants[type].contentTag,
+        dismissible,
         type,
         status,
+        hidden,
         ariaText: input.ariaText || '',
+        ariaLabelClose: input.ariaLabelClose || '',
         htmlAttributes: processHtmlAttributes(input),
-        renderBody: input.renderBody,
         mainClass: [`${type}-notice`, `${type}-notice--${status}`, input.class],
         headingClass: `${type}-notice__status`,
         contentClass: `${type}-notice__content`
     };
 }
+function getTemplateData(state) {
+    return state;
+}
+
+function init() {
+    observer.observeRoot(this, ['hidden'], (hidden) => {
+        this.processAfterStateChange(hidden);
+    });
+}
+/**
+ * Common processing after data change via both UI and API
+ * @param {Boolean} hidden
+ */
+function processAfterStateChange(hidden) {
```

Comment:
I think you can leave out the param and use `this.state.hidden` instead. Make sure to update the jsdoc too.

Reply:
done, thanks for the catch